modules/ROOT/pages/api-groups-sla-tiers.adoc

```diff
@@ -0,0 +1,46 @@
+= SLA Tiers For API Groups
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can specify service-level agreement (SLA) tiers to API Groups instances to limit the number of requests an application (user) can make to the APIs grouped in an API Group. SLAs enable you to protect your API Group instances and help manage network traffic to your APIs.
+
+The SLA tier definition also determines whether access to the API Groups instance requires your approval. You can configure manual or automatic approvals to requests.
+
+== SLA Types
+
+SLA tiers are of two types: 
+
+* Default SLAs
+* Individual API SLAs
+
+== Default 
+
+The default limit is set by the Group Administrator and applies to all API instances. If you add a new API instance to an API group instance, the default limit SLA configured at the group instance level applies, unless an individual API SLA tier set by the Group Administrator. 
+
+== Individual API SLAs
+
+As a Group Administrator, you can define individual limits for API instances within an API Group instance.  If you have configured a default SLA limit at the API Group instance level and an individual SLA limit at the API instance level, the individual SLA takes precedence over the default limit. 
```

Comment:
I think none of you are correct.

@priyaayyar the individual limits do not `take precedence over the default limit`, there completely `override` the default limit. I would also modify some wording because there is no such thing as `Individual API SLAa`, there are individual limits for a specific API in an SLA Tier.

What I think @chiragnarangmulesoft is trying to say (and I agree), is that we should also comment to the user that he can copy a tier created by the API owner and use it as an individual limit for an API (but it is just a copy, they are no linked in any way). This is how you can do it:
<img width="1672" alt="Screen Shot 2020-05-21 at 6 25 53 PM" src="https://user-images.githubusercontent.com/12700293/82608181-8f4cea80-9b90-11ea-8b2b-87311dca39fa.png">

Reply:
Done! Thanks!